CMakeLists.txt

```diff
@@ -0,0 +1,202 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotont_support)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED)
+
+## System dependencies are found with CMake's conventions
```

Comment:
Remove all the unnecessary commented junk. But don't remove "catkin_package(..." while you are doing that.